git-glossary.md

```diff
@@ -59,10 +59,12 @@ and if approved will be merged into the organization repository.
 A collection of [references](#references) to commit objects and an [object database](#object-database) of commit objects. 
 
 
-## reference
-A string name that points to a commit. A reference also includes information about where the [commit](#commit) is located.
-For the [fork/clone development workflow](fork-clone-workflow.md) the commit will be in one of three locations. 
+## remote
+A string name that points to a repository. The value associated with a remote is the URL used to access the remote repository.
+A remote serves as a short alias for a longer URL. 
+For the [fork/clone development workflow](fork-clone-workflow.md) a repository will be in one of three locations. 
```

Comment:
> a repository will be in one of three locations.

Can we change the list below this line to start with the locations and then describe how to access it? (Alternatively we could use another text instead of the above)